static/panes/compiler.ts

```diff
@@ -2881,8 +2827,7 @@ export class Compiler extends MonacoPane<monaco.editor.IStandaloneCodeEditor, Co
         const are = unwiseOptions.length === 1 ? ' is ' : ' are ';
         const msg = options + names + are + 'not recommended, as behaviour might change based on server hardware.';
 
-        // @ts-ignore
-        if (optionsArray.some(opt => opt === '-flto') && !this.filters.state.binary && !wasCmake) {
+        if (optionsArray.some(opt => opt === '-flto') && !this.filters.isSet('binary') && !wasCmake) {
```

Comment:
is this a smart thing to do? this wouldn't detect typos for filternames

Reply:
I agree here but the toggles.ts are generic though in this case, they don't necessarily have to be related to the filters.

Reply:
My TS knowledge isn't strong enough to "template" the Toggles, but I'll try ...

Reply:
per discussion in Discord I'm stepping back from this until I understand TS more! Coming soon™